repo2docker/buildpacks/conda/freeze.py

```diff
@@ -19,30 +19,22 @@
 from ruamel.yaml import YAML
 
 
-DOCKER_IMAGE = "condaforge/mambaforge:4.9.2-5"
-# set mamba and/or conda versions
-# if needed to differ from what's in the image
-MAMBA_VERSION = ""
-CONDA_VERSION = ""
-
 HERE = pathlib.Path(os.path.dirname(os.path.abspath(__file__)))
 
 ENV_FILE = HERE / "environment.yml"
-FROZEN_FILE = os.path.splitext(ENV_FILE)[0] + ".frozen.yml"
+FROZEN_FILE = os.path.splitext(ENV_FILE)[0] + ".lock"
 
 ENV_FILE_T = HERE / "environment.py-{py}.yml"
-FROZEN_FILE_T = os.path.splitext(ENV_FILE_T)[0] + ".frozen.yml"
 
 yaml = YAML(typ="rt")
 
 
-def freeze(env_file, frozen_file):
-    """Freeze a conda environment.yml
+def freeze(env_file, frozen_file, platform="linux-64"):
```

Comment:
This switch to conda-lock removes a significant barrier to supporting arm via setting this to `platform=linux-aarch64`